### DISCUSSION TITLE
Suggestions for easier Stable to Lazer transition for players

### DISCUSSION THREAD

**Discussion by Oszku** [ppy/osu#21860] [Ideas]
These were bothering me after playing laser for a while

1. Add "(Turn off if you want to use an external tablet driver)" or something of that kind to the High precision mouse setting when you hover over it so people won't be confused why their tablet isn't working properly  (like here #21874)

2. If an user selects "Classic Defaults" in the first run setup wizard change the "Deselect all mods" button from backspace to "1" so people won't be confused why their muscle memorized button isn't working (like here #18389)

3. Clear the search bar in settings after closing the settings

4. Adding the ability to hide in-game leaderboards by pressing TAB

5. Map pressing escape after the map ends to go to result screen instad of song select

**Comment by Oszku** [comment]


**Reply by mk56-spn** [reply]
Please update the original post instead of adding addendums like this. It makes it easier to track and reduces spam

**Comment by daixiansen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
